cpp/boot_notification.sh

```diff
@@ -0,0 +1,9 @@
+#!/bin/bash
+set -o errexit -o nounset
+
+
+# The following command does not return until the system is fully up and running!
+/usr/bin/systemctl is-system-running --wait
+
+/usr/local/bin/send_email --recipients=ixtheo-team@ub.uni-tuebingen.de --subject='One of our systems just booted up!' \
```

Comment:
Please put the hostname into the subject. It's just easier to keep the overview in our mailboxes.

Reply:
Done!